node-definitions/freebsd/FreeBSD.yaml

```diff
@@ -0,0 +1,66 @@
+id: FreeBSD
+general:
+  nature: server
+  description: FreeBSD with "nuage-init" (a limited version of cloud-init).
+  read_only: false
+device:
+  interfaces:
+    has_loopback_zero: true
+    physical:
+      - em0
```

Comment:
Why not add multiple NICs like you did with OpenBSD?  I think at least two would be useful.

Reply:
Updated!